src/modules/theme/constants/helpers.js

```diff
@@ -10,3 +10,17 @@ export default {
     text-decoration: none;
   `
 }
+
+export const textTruncator = (text: string, numOfWordsAllowed: number): string => {
```

Comment:
This does not belong here.

Reply:
I'll move it the `common` utils, What do you think?

Reply:
Sounds good